WebView2APISample/README.md

```diff
@@ -25,7 +25,7 @@ Clone the repository and open the solution in Visual Studio. WebView2 is already
 - Clone this repository
 - Open the solution in Visual Studio 2019**
 - Set the target you want to build (Debug/Release, x86/x64/ARM64)
-- Build the project file: `WebView2APISample.vcxproj`
+- Build the solution file: WebView2APISample.sln
```

Comment:
```suggestion
- Build the solution file: `WebView2APISamples.sln`
```

I will fix the copy in ADO too.

Reply:
should this be `WebView2APISamples.vcxproj` instead?

Reply:
If we want to scope to win32 sample project but the collective solution, we can just revert the change here
```suggestion
- Build the project file: `WebView2APISample.vcxproj`
```

Reply:
I just run the script again. and this already got updated. should I still change it to sln?

Reply:
Let's not worry about it then. Thanks!